GitUI/CommandsDialogs/FormPush.Designer.cs

```diff
@@ -104,7 +104,7 @@ private void InitializeComponent()
             this.ForcePushBranches.Name = "ForcePushBranches";
             this.ForcePushBranches.Size = new System.Drawing.Size(79, 17);
             this.ForcePushBranches.TabIndex = 1;
-            this.ForcePushBranches.Text = "&Force Push";
+            this.ForcePushBranches.Text = "&Force push";
```

Comment:
(if this is the site close to force-with-lease)
`F` should be used with force-with-lease.
There was an issue requesting that --force was removed completely. I still prefer to keep it but usage should not be prominent
```suggestion
            this.ForcePushBranches.Text = "Force push";
```

Reply:
`F` is used with force-with-lease now (and `O` for "F&orce push" because it would look incomplete).